tests/ProductsTest.php

```diff
@@ -73,6 +73,23 @@ public function testProductsXml()
         $this->assertXmlStringEqualsXmlString($sampleXml, $outputXml);
     }
 
+    public function testProductsDeletedXml()
+    {
+        $document = new Document('TestCatalog');
+
+        foreach (['Product', 'Variation'] as $example) {
```

Comment:
I confused 'Product' and `Product` at first glance, so `['PRODUCT', 'VARIATION']` instead of the `strtoupper()` would help differentiate them when skimming.

Or perhaps drop the loop and keep it simple:

```
$element = new Product('PRD123');
$element->setDeleted();
$document->addObject($element);

$element = new Product('VAR123');
$element->setDeleted();
$document->addObject($element);
```